File: Python/Assignments/coins.py
import math

print("Please enter the amount of money to convert:")

inp_num_dollars = float("{:.0f}".format(float(input("# of dollars: "))))
inp_num_cents = float("{:.2f}".format(float(input("# of cents: "))))
#print ("inp_num_dollars =", inp_num_dollars, "\n", "inp_num_cents",inp_num_cents,"\n")

total_money = inp_num_dollars + inp_num_cents/100
#print("total_money", total_money)


dollars_to_quarters = float(total_money) // 0.25
c = total_money %.25
leftover_dimes= c // 0.10
c = c%.10
leftover_nickles=c // 0.05
c = c%.05
#print("c", c)
c = round(c,2)
#print("c",c)
leftover_pennies = c / 0.01
#print("leftover_pennies =", leftover_pennies)

print("The coins are", int(dollars_to_quarters) ,"quarters,", int(leftover_dimes) ,"dimes,", int(leftover_nickles),"nickels" ,"and" ,int(leftover_pennies), "pennies")

File: C++/Assignments/coins.cpp
# include <iostream>
using namespace std;

int main()
{
	double inp_dollars, inp_cents, total_quaters,total_dimes,total_nickles,total_pennies;
	int c,total;
	
	cout << "Please enter the amount of money to convert:" <<endl;
	cout << "# of dollars: ";
	cin >> inp_dollars;
	cout << "# of cents: ";
	cin >> inp_cents;
	total = (inp_dollars + inp_cents/100)*100;
	cout << "toatal" << total;
	total_quaters = total / 25;
	c = total % 25;
	total_dimes = c / 10;
        c = c % 10;
        total_nickles = c/5;
	c =c% 5;
        total_pennies = c/1;
        cout << "The coins are "<<total_quaters<<" quarters, "<<total_dimes<<" dimes, "<<total_nickles<<" nickels and "<<total_pennies<< " pennies"<<endl;

}
